app/models/direct_file_data.rb

```diff
@@ -515,22 +524,22 @@ def fed_w2_state=(value)
     write_df_xml_value(__method__, value)
   end
 
-  def primary_blind
-    create_or_destroy_df_xml_node(__method__, true, 'VirtualCurAcquiredDurTYInd')
-    write_df_xml_value(__method__, "X")
+  def set_primary_blind
+    create_or_destroy_df_xml_node(:primary_blind, true, 'VirtualCurAcquiredDurTYInd')
+    write_df_xml_value(:primary_blind, 'X')
   end
 
-  def spouse_blind
-    create_or_destroy_df_xml_node(__method__, true, 'VirtualCurAcquiredDurTYInd')
-    write_df_xml_value(__method__, "X")
+  def set_spouse_blind
+    create_or_destroy_df_xml_node(:spouse_blind, true, 'VirtualCurAcquiredDurTYInd')
+    write_df_xml_value(:spouse_blind, 'X')
```

Comment:
refactored here because the `primary_blind` and `spouse_blind` xml readers were being overridden by this custom setter. Now this method is more clear about the purpose and also allows us to easily access the value in the `is_primary_blind?` and `is_spouse_blind` methods

Reply:
@jachan the refactor promised -- pls let me know if you have concerns. Unfortunately the implementation is split between multiple commits :( but if you just look at the changes in this file (and maybe the nj intakes factory) you should be caught up!